### PR TITLE
Add EvilYep card info

### PR DESCRIPTION
**Pull request by EvilYep**
Added my card to index.html

Thanks for the tutorial !

### CODE REVIEW

**Review by Preet-Sojitra — APPROVED** [review]
LGTM 👍 
Your PR will be merged soon.
Have a nice day :)

**Comment by EvilYep** [comment]
Thanks ! Nice day to you too !

File: index.html [hunk 1]
```diff
@@ -115,6 +115,36 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ EvilYep's Contributor card START ________  -->
+        <div class="card">
+          <p class="name">💾 EvilYep</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/EvilYep" target="_blank">Github</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://github.com/EvilYep" target="_blank">@EvilYep</a>
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,36 @@ <h2>A project for first-time contributions</h2>
...
+            <br>
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/gmoneret/" target="_blank">LinkedIn</a>
```
Comment (TenTraicion) [inline index.html:126]:
```suggestion
            <a href="https://www.linkedin.com/in/gmoneret/" target="_blank">@gmoneret</a>
```

File: index.html [hunk 3]
```diff
@@ -115,6 +115,36 @@ <h2>A project for first-time contributions</h2>
...
+            <br>
+            <i class="fas fa-link"></i>
+            <a href="https://www.malt.fr/profile/guillaumemoneret" target="_blank">Malt</a>
```
Comment (TenTraicion) [inline index.html:129]:
```suggestion
            <a href="https://www.malt.fr/profile/guillaumemoneret" target="_blank">@guillaumemoneret</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Thanks for your contribution, @EvilYep. Kindly do the recommended changes and commit. Your PR will be merged soon!

**Comment by EvilYep** [comment]
Thanks ! I did push the sugggested changes to my branch. Should I "resolve conversation" ? Is it the proper way ?

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@EvilYep, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Comment by EvilYep** [comment]
@TenTraicion there https://github.com/Syknapse/Contribute-To-This-Project/pull/1705 you directedd me back here...

Well I'm really sorry, I didn't realized the changes you ssuggested would "reveal" my name in the links I provided. I'm not too comfortable with it being all around.

Is there really no way to delete my last 2 links just under the title, just leaving the Github one ?

**Comment by TenTraicion** [comment]
@EvilYep Thanks for commenting on the original PR. Yeah it is possible to remove extra social links, but one is mandatory. Kindly confirm the changes, one of our maintainer will do that for you.

**Comment by TenTraicion** [comment]
you can also seek help from our discord server: https://discord.gg/tWkvS4ueVF

**Comment by EvilYep** [comment]
@TenTraicion Thanks for the answer

I'm sorry I'm a real beginner in the Git flow, how would I "confirm" the changes ? By making a new PR ?
Or just confirming here that I woulld like only my first link (the Github one) to appear ?

**Comment by EvilYep** [comment]
Oh ok going to discord right now

**Comment by TenTraicion** [comment]
@EvilYep please wait! as i am out of workplace right now one of our maintainer will shortly do the changes. Thanks!

**Comment by EvilYep** [comment]
@TenTraicion Ok thanks very much ! Please accept my apologies for the waste of time

**Comment by TenTraicion** [comment]
> @TenTraicion Ok thanks very much ! Please accept my apologies for the waste of time

no sweat! we will do the changes asap and let you know! :)

**Comment by TenTraicion** [comment]
@EvilYep done via #1705. Happy Coding. Sorry for being Late though!

**Comment by EvilYep** [comment]
Thanks a lot !

**Pull request merged**